components/media/AudioPlayer.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <!--
+    `key` attribute is needed to force replacement of the entire audio element
+    when `src` updates as `source` elements may not have their `src` attribute
+    updated after render.
+  -->
+  <audio
+    :key="src"
+    :controls="controls"
+    :autoplay="autoplay"
+    :loop="loop"
+    data-qa="audio player"
```

Comment:
add:
```
    class="w-100"
```
it makes the display much nicer as well as improving accessibility of the seek bar, otherwise it's really hard to jump to a specific section of longer audio files.

Reply:
Done